workspace/src/path_planning/centerline_objects_path_planner/centerline_objects_path_planner/centerline_objects_path_planner.py

```diff
@@ -49,15 +49,32 @@
 import json
 
 
-class PathPlanningNode(Node):
+class ConePathPlannerNode(Node):
+    """A path planning node based on navigating a path defined by cones on either side.
+
+    This path planner receives a list of green and red cone locations on either side of the vehicle. It then defines a short-term local path (visualizable) for the control node to follow.
+
+    Attributes:
+        lookahead: How far ahead of the car the target point should be.
+        vis: Whether or not to visualize the path planning.
+        green_cones: A list of green cone locations, to be kept to the left of the vehicle.
+        red_cones: A list of red cone locations, to be kept to the right of the vehicle.
+    """
+
     def __init__(self):
-        super().__init__("path_planning_node")
+        """initialize the Path planning Node.
+
+        Initialize the path planning node, and set up the publishers / subscribers.
+        """
+        super().__init__("centerline_objects_path_planner_node")
 
         # update frequency of this node
         self.freq = 10.0
 
         # READ IN SHARE DIRECTORY LOCATION
```

Comment:
yelling

Reply:
Yeah if you look at my commit history it says "add non-descriptive doc strings"... I mostly just did this because I thought they should be there, even though for a lot of the nodes I don't really have a good enough understanding for what they do (esp path planning, idk the type of message it publishes, how it calculates that, etc.). I can go back and fix this... but not sure it will actually be accurate in regards to what is actually going on

Reply:
I'm not gonna fix this in this PR, but can create an issue for it and do it in a future PR. I'm just hoping to get this merged so that I can also add the waypoints path planner and MPC controller in this structure of the repo, so that Harry can do physical testing.
(unless this is important enough in your opinion that I should fix it here...)